src/lib.rs

```diff
@@ -45,13 +45,13 @@ pub use set::*;
 pub mod raw {
     //! Exposes raw syscalls that simply return a `usize` instead of a `Result`.
 
-    pub use super::syscall::syscall0;
-    pub use super::syscall::syscall1;
-    pub use super::syscall::syscall2;
-    pub use super::syscall::syscall3;
-    pub use super::syscall::syscall4;
-    pub use super::syscall::syscall5;
-    pub use super::syscall::syscall6;
+    pub use super::syscall::raw_syscall0;
```

Comment:
Small, but important, build error: `raw_syscallX` -> `syscallX`

Changing `syscalls::raw::syscallX` to take a `usize` instead of `Sysno` is also technically a breaking change, so I think I'll release a new major version of the library.

Reply:
Seems I previously only changed the function names on the x86-64 arch, and this is why its currently the only arch that's correctly building.

I also saw your other comment, so now I've reverted x86-64 instead of changing the other archs (revert the function names on x86-64 to `syscall[0-6]` instead of `raw_syscall[0-6]`)